x-pack/test/functional/apps/dataset_quality/dataset_quality_flyout.ts

```diff
@@ -118,6 +118,120 @@ export default function ({ getService, getPageObjects }: DatasetQualityFtrProvid
+      const flyoutBodyContainer = await testSubjects.find(
+        PageObjects.datasetQuality.testSubjectSelectors.datasetQualityFlyoutBody
+      );
+      await PageObjects.datasetQuality.setDatePickerLastXUnits(flyoutBodyContainer, 3, 'd');
+
+      // Index 2 doc 2 days ago
+      const time2DaysAgo = Date.now() - 2 * 24 * 60 * 60 * 1000;
+      await synthtrace.index(
+        getLogsForDataset({
+          to: time2DaysAgo,
+          count: 2,
+          dataset: apacheAccessDatasetName,
+          isMalformed: false,
+        })
+      );
+
+      // Index 5 degraded docs 2 days ago
+      await synthtrace.index(
+        getLogsForDataset({
+          to: time2DaysAgo,
+          count: 5,
+          dataset: apacheAccessDatasetName,
+          isMalformed: true,
+        })
+      );
+
+      await PageObjects.datasetQuality.refreshFlyout();
+      const summaryAfter = await PageObjects.datasetQuality.parseFlyoutKpis();
+
+      expect(parseInt(summaryAfter.docsCountTotal, 10)).to.be.greaterThan(
+        parseInt(summaryBefore.docsCountTotal, 10)
+      );
+
+      expect(parseInt(summaryAfter.degradedDocs, 10)).to.be.greaterThan(
+        parseInt(summaryBefore.degradedDocs, 10)
+      );
+
+      expect(parseInt(summaryAfter.size, 10)).to.be.greaterThan(parseInt(summaryBefore.size, 10));
+      expect(parseInt(summaryAfter.services, 10)).to.be.greaterThan(
+        parseInt(summaryBefore.services, 10)
+      );
+      expect(parseInt(summaryAfter.hosts, 10)).to.be.greaterThan(parseInt(summaryBefore.hosts, 10));
+    });
+
+    it('shows the right number of services', async () => {
+      const apacheAccessDatasetName = 'apache.access';
+      const apacheAccessDatasetHumanName = 'Apache access logs';
+      await PageObjects.datasetQuality.openDatasetFlyout(apacheAccessDatasetHumanName);
+
+      const summaryBefore = await PageObjects.datasetQuality.parseFlyoutKpis();
+      const testServices = ['test-srv-1', 'test-srv-2'];
+
+      // Index 2 docs with different services
+      const timeNow = Date.now();
+      await synthtrace.index(
+        getLogsForDataset({
+          to: timeNow,
+          count: 2,
+          dataset: apacheAccessDatasetName,
+          isMalformed: false,
+          services: testServices,
+        })
+      );
+
+      await PageObjects.datasetQuality.refreshFlyout();
+      const summaryAfter = await PageObjects.datasetQuality.parseFlyoutKpis();
+
+      expect(parseInt(summaryAfter.services, 10)).to.eql(
+        parseInt(summaryBefore.services, 10) + testServices.length
+      );
+    });
+
+    it('goes to log explorer for degraded docs when show all is clicked', async () => {
+      const apacheAccessDatasetName = 'apache.access';
+      const apacheAccessDatasetHumanName = 'Apache access logs';
+      await PageObjects.datasetQuality.openDatasetFlyout(apacheAccessDatasetHumanName);
+
+      // Get reference to "Show all" anchor element manually navigate to the href to avoid opening a new tab
```

Comment:
is this comment still relevant? from what I see we are switching tabs anyway

Reply:
No, it was an old comment. Thanks for spotting. I've removed it.